### PR TITLE
Get Auto Failover Settings operation

### PR DESCRIPTION
**Pull request by mihaitusa**
Couchbase - Added Get Auto Failover Settings operation && bring the integration close to this days && refactoring

Signed off by Tusa Mihai: tusa.mihai@gmail.com

### CODE REVIEW

**Comment by mihaitusa** [comment]
@tethryus Please remove java 7 build... believe me... no longer needed... :smile:

**Comment by tethryus** [comment]
jdk 7 removed 👍

File: src/main/java/io/cloudslang/content/couchbase/actions/buckets/DeleteBucket.java [hunk 1]
```diff
@@ -180,4 +208,11 @@
             return getFailureResultsMap(exception);
         }
     }
+
+    public static void main(String[] args) {
```
Comment (Bonczidai) [inline src/main/java/io/cloudslang/content/couchbase/actions/buckets/DeleteBucket.java]: hmm.. should the main methods be merged into master?

Comment (mihaitusa) [reply src/main/java/io/cloudslang/content/couchbase/actions/buckets/DeleteBucket.java]: :)

**Comment by mihaitusa** [comment]
@tethryus Now it's up to you...

**Pull request merged**